flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/api/ExecutionConfigOptions.java

```diff
@@ -174,16 +174,38 @@
 							" is set true, its value must be positive.");
 
 	// ------------------------------------------------------------------------
-	//  State Options
+	//  Emit Strategy Options
 	// ------------------------------------------------------------------------
-	public static final ConfigOption<String> SQL_EXEC_STATE_TTL =
-			key("sql.exec.state.ttl")
-					.defaultValue("-1 ms")
-					.withDescription("Specifies a minimum time interval for how long idle state " +
-							"(i.e. state which was not updated), will be retained. State will never be " +
-							"cleared until it was idle for less than the minimum time, and will be cleared " +
-							"at some time after it was idle. Default is never clean-up the state.\n" +
-							"NOTE: Cleaning up state requires additional overhead for bookkeeping.");
+
+	public static final ConfigOption<Boolean> SQL_EXEC_EMIT_EARLY_FIRE_ENABLED =
```

Comment:
why are you adding these configs?

Reply:
These configuration is get/set by old TableConfig in Blink planner which is not contained by TableConfig in Api  module. After remove old TableConfig in Blink planner, I move the configuration to ExecutionConfigOptions.

Reply:
@wuchong Please review the configuration message to check whether it describes correctly.

Reply:
Remove those configuration from ExecutionConfigOptions.